nilearn/plotting/tests/test_img_plotting/test_plot_roi.py

```diff
@@ -54,6 +54,12 @@ def test_plot_roi_view_types(
         cut_coords=cut_coords,
         **kwargs,
     )
+
+    for _ in range(len(recwarn)):
```

Comment:
I'm a bit puzzled. Why do we need this call at this place in particular ? Can you explain ?

Reply:
This relates to the previous comment.

> Should we add a test to enforce the new behavior ?

The new behavior is to NOT throw a that specific warning so in each case, I loop through all the warning and check their type and message to make sure they are not of the one we want to avoid.

Reply:
The recwarn is a pytest fixture to catch warnings.

I don't think we have a consistent way to check for this in the test suite.

Reply:
OK, I see. 
Maybe adding a comment in the code would be useful for people to understand why this was added ?

Reply:
I actually put that in a separate test because there was not point in testing this with the parametrization.